.github/actions/is-admin/README.md

```diff
@@ -0,0 +1,53 @@
+## About
+
+GitHub Action to check if someone is admin of the given GitHub repository.
+
+* [Usage](#usage)
+  * [Configuration](#configuration)
+* [Customizing](#customizing)
+  * [inputs](#inputs)
+  * [outputs](#outputs)
+
+## Usage
+
+### Configuration
+
+Given the CI GitHub action:
+
+```yaml
+---
+name: Is Admin Example
+on:
+  issues:
+    types: [opened]
+jobs:
+  run-if-admin:
+    runs-on: ubuntu-latest
+    steps:
+      - id: is_admin
+        uses: elastic/apm-pipeline-library/.github/actions/is-admin@current
+        with:
+          username: ${{ github.event.issue.user.login }}
+          token: ${{ secrets.PAT_TOKEN }}
```

Comment:
I need to double check since in some cases the permissions were not enough to access some specific Rest API. But let me double check that. 
On the other hand, we are safe it's just an example.

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2144/commits/ebdfd475433a4a9cd4691091aa65fa67d21c5d0d worked like a charm, so I'll then use the default secrets then